deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -33,6 +34,7 @@ const LEGACY_URI_NEEDING_TRAILING_SLASH = new RegExp(
 
 const CONTENT_DEVELOPMENT_DOMAIN = ".content.dev.mdn.mozit.cloud";
 
+const require = createRequire(import.meta.url);
 const REDIRECTS = require("./redirects.json");
```

Comment:
Rather than creating a require, why don't we use `fs.readFile`?
```suggestion
const REDIRECTS = JSON.parse(
  await fs.readFile(new URL('./redirects.json', import.meta.url))
);
```

Reply:
I wanted to avoid the risk that fs.readFile + JSON.parse may unexpectedly behave different in AWS Lambda. Once we moved to GCP by end of January 2023 (current plan), these lambdas will be history. :/